packages/riipen-ui/src/components/Button.jsx

```diff
@@ -400,6 +401,88 @@ class Button extends React.Component {
             border-color: ${theme.palette.negative.main};
           }
 
+          .inverted {
+            background-color: white;
+            color: ${theme.palette.text.primary};
+            overflow: hidden;
+            position: relative;
+          }
+          .inverted::after {
+            border-radius: ${theme.shape.borderRadius.md};
+            border: 1px solid rgba(0, 0, 0, 0.23);
+            bottom: 0;
+            content: "";
+            left: 0;
+            position: absolute;
+            right: 0;
+            top: 0;
+          }
+          .inverted:hover::before {
+            background-color: ${theme.palette.grey[600]};
+            bottom: 0;
+            content: "";
+            left: 0;
+            opacity: 0.1;
+            position: absolute;
+            right: 0;
+            top: 0;
+          }
+
+          .inverted-primary {
+            color: ${theme.palette.primary.main};
+          }
+          .inverted-primary:hover::before {
+            background-color: ${theme.palette.primary.main};
+          }
+          .inverted-primary::after {
+            border-color: ${theme.palette.primary.main};
+          }
+          .inverted-secondary {
+            color: ${theme.palette.secondary.main};
+          }
+          .inverted-secondary:hover::before {
+            background-color: ${theme.palette.secondary.main};
+          }
+          .inverted-secondary::after {
+            border-color: ${theme.palette.secondary.main};
+          }
+          .inverted-tertiary {
+            color: ${theme.palette.tertiary.main};
+          }
+          .inverted-tertiary:hover::before {
+            background-color: ${theme.palette.tertiary.main};
+          }
+          .inverted-tertiary::after {
+            border-color: ${theme.palette.tertiary.main};
+          }
+          .inverted-white {
+            color: ${theme.palette.common.black};
+          }
+          .inverted-white:hover::before {
+            background-color: ${theme.palette.grey[100]};
+          }
+          .inverted-white::after {
+            border-color: ${theme.palette.common.black};
+          }
```

Comment:
```suggestion
          .inverted-black {
            color: ${theme.palette.common.black};
          }
          .inverted-black:hover::before {
            background-color: ${theme.palette.grey[100]};
          }
          .inverted-black::after {
            border-color: ${theme.palette.common.black};
          }
```

Reply:
Would need to add `black` to `propTypes:{color}` also

Reply:
/ For `inverted-white`, I think it should have `{backgroundColor: black, color: white}`

Reply:
I dont think the use case for this is very large, I am going to remove it for now and let someone build it when the use case presents itself